docs/apache-airflow-providers-qdrant/connections.rst

```diff
@@ -0,0 +1,58 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection:qdrant:
+
+Qdrant Connection
+===================
+
+The `Qdrant <https://qdrant.tech/>`__ connection type enables access to Qdrant clusters.
+
+Default Connection IDs
+----------------------
+
+The Qdrant hook use the ``qdrant_default`` connection ID by default.
+
+Configuring the Connection
+--------------------------
+
+Host (optional)
+    Host of the Qdrant instance to connect to.
+
+API key (optional)
+    Qdrant API Key for authentication.
+
+Port (optional)
+    REST port of the Qdrant instance to connect to. Defaults to ``6333``.
+
+URL (optional)
+    URL of the Qdrant instance to connect to. If specified, it overrides the ``host`` and ``port`` parameters.
+
+GRPC Port (optional)
+    GRPC port of the Qdrant instance to connect to. Defaults to ``6334``.
+
+Prefer GRPC (optional)
+    Whether to use GRPC for custom methods. Defaults to ``False``.
+
+HTTPS (optional)
+    Whether to use HTTPS for requests. Defaults to ``False``.
```

Comment:
Why default false?

Reply:
Since local Qdrant  deployments(at localhost) don't usually have TLS configured.
Most people start off with a local instance.

Reply:
Also, if an API key is set, `HTTPS` defaults to true.
`qdrant_client` handles this internally.

Reply:
Nice. Good catch @eladkal , thanks for insightful retrospection @Anush008 :)

Reply:
Would be nice to explain this inference in here.

Reply:
https://github.com/apache/airflow/pull/36805/commits/08d78845531a3aedd1eb91c8efa45e040dec3c1e